saleor/static/dashboard-next/customers/views/CustomerAddresses.tsx

```diff
@@ -140,140 +141,99 @@ const CustomerAddresses: React.FC<CustomerAddressesProps> = ({ id }) => {
-                                        }
-                                      })
-                                    }
-                                  />
-                                )}
-                              />
-                              <Route
-                                path={customerAddressRemovePath(
-                                  ":customerId",
-                                  ":addressId"
-                                )}
-                                render={({ match }) => (
-                                  <ActionDialog
-                                    open={!!match}
-                                    variant="delete"
-                                    title={i18n.t("Remove Address")}
-                                    confirmButtonState={
-                                      removeAddressTransitionState
-                                    }
-                                    onClose={closeModal}
-                                    onConfirm={() =>
-                                      removeCustomerAddress({
-                                        variables: {
-                                          id: decodeURIComponent(
-                                            match.params.addressId
-                                          )
-                                        }
-                                      })
-                                    }
-                                  >
-                                    <DialogContentText>
-                                      {i18n.t(
-                                        "Are you sure you want to remove this address from users address book?"
-                                      )}
-                                    </DialogContentText>
-                                  </ActionDialog>
-                                )}
-                              />
-                              <Route
-                                path={customerAddressPath(
-                                  ":customerId",
-                                  ":addressId"
-                                )}
-                                render={({ match }) => (
-                                  <CustomerAddressDialog
-                                    address={maybe(() =>
-                                      customerData.data.user.addresses.find(
-                                        addr =>
-                                          addr.id ===
-                                          decodeURIComponent(
-                                            match.params.addressId
-                                          )
-                                      )
-                                    )}
-                                    confirmButtonState={
-                                      updateAddressTransitionState
+                            <CustomerAddressDialog
+                              address={undefined}
```

Comment:
Does it need be explicit? Mb can be set as not required on `CustomerAddressDialog` if so?